src/pageObject.js

```diff
@@ -1,16 +1,40 @@
 'use strict';
 
 class Header {
-
-};
+  get logo() {
+    return 'Conduit logo';
+  }
+}
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+  }
+  header = new Header();
 
-};
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
+}
 
 class ArticlePage extends PageObject {
+  constructor(url, slug) {
+    super(url);
+    this.url = url + slug;
+  }
 
-};
+  get commentButton() {
+    return '[Publish comment] button';
+  }
+
+  clickOnCommentButton() {
+    return `Click on the ${this.commentButton}`;
+  }
+
+  assertPageOpened() {
+    return `The ${this.url} is opened`;
+  }
+}
 
 module.exports = {
   Header,
```

Comment:
You are only exporting the Header class, but the PageObject and ArticlePage class are not being exported.

Reply:
Fixed in 'Add comment'